package.json

```diff
@@ -10,5 +10,14 @@
     "lint": "cd frontend && npm run lint",
     "postinstall": "npm run build && npm run deploy",
     "start": "cd frontend && npm start"
+  },
+  "dependencies": {
+    "install": "^0.12.2",
+    "prop-types": "^15.7.2",
+    "react-accessible-accordion": "^3.0.0",
+    "react-accordeon": "^1.1.1",
+    "react-collapse": "^4.0.3",
+    "react-collapsible": "^2.6.0",
+    "eslint": "^5.16.0"
```

Comment:
We don't want to install dependencies at this level, so probably want to back these changes out. Everything should go in the frontend/package.json file, and these aren't getting used at all.